### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by zentyz** [insilichem/gaudi#31]
Hi everyone or anyone,

I desperately need help.

I was trying to install in both py2 and py3 environment but both gives me an error.

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - gaudi -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0']

Your python: python=2.7

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.

The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.35=0
  - feature:|@/linux-64::__glibc==2.35=0

Your installed version is: 2.35

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - gaudi -> python[version='2.7.*|>=2.7,<2.8.0a0']

Your python: python=3.10

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.

The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.35=0
  - feature:|@/linux-64::__glibc==2.35=0

Your installed version is: 2.35

**Comment by zentyz** [comment]
Yes. I have installed chimera and also went into the python 2.7 env. right now i get this message.

conda create -n insilichem -c omnia -c salilab -c insilichem -c conda-forge -c bioconda -c tpeulen gaudi
Collecting package metadata (current_repodata.json): done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: \ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                          

UnsatisfiableError: The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.35=0
  - gaudi -> openmm -> __glibc[version='>=2.17|>=2.17,<3.0.a0']

Your installed version is: 2.35

**Comment by zentyz** [comment]
Hi,

Thank you so much for your prompt reply.

I got it to somehow work, but i am wondering if the plugin will affect the results you have obtained in your publication on ACS omega "Simple Coordination Geometry Descriptors Allow to Accurately Predict Metal-Binding Sites in Proteins". It is exactly what we need for our research.

**Comment by jaimergp** [comment]
`imp` is only required in the `ligscore` objective, IIRC.

I am no longer involved in research, but feel free to connect with @JeanDidier for more work in that direction.
